app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/UIHelper.cs

```diff
@@ -110,9 +110,9 @@ public static IEnumerable<Face> CalculateFaceRectangleForRendering(IList<Detecte
         {
             var imageWidth = imageInfo.Item1;
             var imageHeight = imageInfo.Item2;
-            float ratio = (float)imageWidth / imageHeight;
-            int uiWidth = 0;
-            int uiHeight = 0;
+            var ratio = (float)imageWidth / imageHeight;
+            var uiWidth = 0;
+            var uiHeight = 0;
```

Comment:
[Nit] A more elegant way is to directly use
`            int uiWidth;`
`            int uiHeight;`

Reply:
Done.